### DISCUSSION TITLE
建议，添加多个快捷键

### DISCUSSION THREAD

**Discussion by QYYE21** [Snipaste/feedback#2717] [General]
我想要添加多个快捷键，希望可以加上这个功能
![Snipaste_2023-09-13_13-14-23](https://github.com/Snipaste/feedback/assets/144876041/c60bec9c-97bc-49e2-8173-9b0664fc6a3f)

**Comment by Geeyun-JY3** [comment]
专业版可以：https://github.com/Snipaste/feedback/issues/2706#issuecomment-1703847513
